tensorrt_llm/executor/utils.py

```
@@ -101,6 +101,9 @@ def submit_sync(self, task: Callable, *args, **kwargs) -> List[Any]:
         return [future.result() for future in futures]
 
     def shutdown(self):
+        print(
+            f"==================================== shutdown ProcessPoolExecutor session"
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)